app/templates/components/forms/events/view/create-discount-code.hbs

```diff
@@ -0,0 +1,94 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <div class="field">
+    <label class="required">{{t 'Discount Code'}}</label>
+    {{input type='text' name='discount_code' value=discount_code}}
+  </div>
+  <div class="field">
+    <label class="required">{{t 'Enter Discount'}}</label>
+    {{ui-radio label=(t 'Amount (US$)') name='discount_type'  value='amount' current='amount' onChange=(action (mut selectedMode))}}
+  </div>
+  {{#if (not-eq selectedMode 'percent')}}
+    <div class="field">
+      {{input type='number' name='discount_amount' min=1}}
+    </div>
+  {{/if}}
+  <div class="field">
+    {{ui-radio label=(t 'Percent (%)') name='discount_type'  value='percent' onChange=(action (mut selectedMode))}}
+  </div>
+  {{#if (eq selectedMode 'percent')}}
+    <div class="field">
+      {{input type='number' name='discount_percent' min=1 max=100 step=0.1}}
+    </div>
+  {{/if}}
+  <div class="field">
+    <label class="required">{{t 'Number of Discount Tickets'}}</label>
+    {{input name='number_of_discount_tickets' type='number' value='10' min='0'}}
+  </div>
+  <div class="grouped inline fields">
+    <label class="required">{{t 'Status'}}</label>
+    <div class="field">
+      {{ui-radio name='status' label='Active' current='active' value='active'}}
+    </div>
+    <div class="field">
+      {{ui-radio name='status' label='Inactive' value='inactive'}}
+    </div>
+  </div>
+  <div class="field">
+    <label class="required">{{t 'Select Ticket(s) applied to the discount code'}}</label>
+    <div class="ui checkbox">
+      {{ui-checkbox label="Select all Ticket types" checked=checked onChange=(action (mut checked))}}
+    </div>
+  </div>
+  <div>
+    <div class="inline field">
+      <label>{{t 'Show more options'}}</label>
```

Comment:
This label should not be defined explicitly, for the checkbox, pass it as you have passed for the above checkboxes please